app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -574,17 +650,31 @@ $after-button-padding-left: govuk-spacing(4);
     right: 0;
 
     @include focus-not-focus-visible {
-      border-bottom: 1px solid govuk-colour("dark-blue");
       border-left: none;
       position: relative;
     }
 
     &:hover {
-      background: govuk-colour("black");
       border-bottom: $pseudo-underline-height solid govuk-colour("mid-grey");
       color: govuk-colour("mid-grey");
     }
 
+    &.gem-c-layout-super-navigation-header__search-toggle-button--blue-background {
+      &:hover {
+        border-bottom: $pseudo-underline-height solid govuk-colour("white");
+        color: govuk-colour("white");
+      }
+    }
+
+    &:focus-visible {
+      &.gem-c-layout-super-navigation-header__search-toggle-button--blue-background {
+        &:hover {
+          border-bottom: $pseudo-underline-height solid $govuk-focus-colour;
```

Comment:
The focus state overhangs a bit on mobile, this is on Android 12 on Galaxy S23. The focus isn't visible much on touch screens but probably worth tweaking still.
<img width="369" alt="Screenshot 2023-10-03 at 19 33 48" src="https://github.com/alphagov/govuk_publishing_components/assets/5007934/19bffa11-60e3-465b-aa98-b57d7d9916db">

Reply:
this shouldn't be an issue now (large styles not on mobile)